net-misc/ntpsec/ntpsec-9999.ebuild

```diff
@@ -164,12 +153,18 @@ src_install() {
 
 	# move doc files to /usr/share/doc/"${P}"
 	use doc && mv -v "${ED}"/usr/share/doc/"${PN}" "${ED}"/usr/share/doc/"${P}"/html
-}
-
-python_install() {
+	local impl
+	for impl in "${_PYTHON_SUPPORTED_IMPLS[@]}"; do
```

Comment:
This is an internal variable, please don't use it. See other comments for how to replace it.